frontend/app/src/people/main/Header.tsx

```diff
@@ -340,7 +340,7 @@ function Header() {
   }, []);
 
   function goToEditSelf() {
-    if (ui.meInfo?.id) {
+    if (ui.meInfo?.id && location.pathname !== `/p/${ui.meInfo.owner_pubkey}/organizations`) {
```

Comment:
can we make it so the `location.pathname` does not contain `/p/${ui.meInfo.owner_pubkey}` because then this would also make it so we don't always get sent back to the org page when we're on our profile page

Reply:
can we also add a test for this aswell

Reply:
here is an example routing test

https://github.com/stakwork/sphinx-tribes/pull/1251/files#diff-41ea9b7fed8cd4323c0aad535fe5a4e7c219ad42341ccd0e5427051e84a0d3ffR163

Reply:
Sure @kevkevinpal I have addressed your first comment.
Will add the test case too.